pkto_cockatrice.xml

```diff
@@ -2369,7 +2370,7 @@ When there are four or more charge counters on Dowsing Machine, sacrifice it.</t
 			<name>Disable</name>
 			<set picURL="https://brooks42.github.io/pkto/cards/Disable.png" rarity="common">PKTO</set>
 			<color>UB</color>
-			<manacost>{U/B}</manacost>
+			<manacost>U{U/B}</manacost>
```

Comment:
UU/B instead of 1U/B huh 🤔

Reply:
Yeah I think counterspells should require blue.

Reply:
https://gatherer.wizards.com/Pages/Card/Details.aspx?multiverseid=126812

;P

Reply:
I think Disable should draw a card if it requires blue. Or at least rummage?

Reply:
![image](https://user-images.githubusercontent.com/2886811/181880387-224aa04c-1fe3-4cb1-b1d2-a72782b026c9.png)

![image](https://user-images.githubusercontent.com/2886811/181880399-b381110e-3fee-4a51-b7b6-35de589ae16e.png)

I think rummaging would be ideal, but it's a lot of text :/

Reply:
Stop citing planar chaos lol, it doesn't count. I think rummaging would be fine to add, but it is a lot of text. Even if it wasn't itty bitty, it's just going to be a lot to process on one card. I think countering a thing for two turns is pretty good already.